x/ccv/provider/keeper/proposal.go

```diff
@@ -116,19 +190,25 @@ func (k Keeper) MakeConsumerGenesis(ctx sdk.Context) (gen consumertypes.GenesisS
 	return gen, nil
 }
 
-// SetConsumerClient sets the clientID for the given chainID
+// SetConsumerClient sets the client ID for the given chain ID
```

Comment:
`SetConsumerClient`, `GetConsumerClient`, and `DeleteConsumerClient` should be move to `keeper.go`. It will make it easier to keep track of the consumer state.